### DISCUSSION TITLE
BUG: Unhandled error in plugin from_transmission: can't compare offset-naive and offset-aware datetimes

### DISCUSSION THREAD

**Discussion by chrismcdowell25** [Flexget/Flexget#3080] [Q&A]
Upgraded and now getting an error with from_transmission.

[crash_report.2021.07.13.113837483299.log](https://github.com/Flexget/Flexget/files/6811256/crash_report.2021.07.13.113837483299.log)

**Comment by timulthyus** [comment]
I've also started getting the same error with from_transmission since upgrading. My crash log looks virtually the same with the only noticeable difference being I'm running python3.9. 

Happy to test / provide logs too should anyone much more knowledgeable than I have any theories on how to fix.

**Comment by marco-russian** [comment]
What is happening with this bug I am now experiencing it

**Comment by gazpachoking** [comment]
It should be fixed. Make sure you have latest flexget, latest transmission-rpc, and if you are using transmission date fields in a template, or if statement to compare to `now` you might need to change to `now.astimezone()`
